runtime/runtime/src/config.rs

```diff
@@ -58,7 +61,25 @@ pub fn total_send_fees(
             }
             Transfer(_) => cfg.transfer_cost.send_fee(sender_is_receiver),
             Stake(_) => cfg.stake_cost.send_fee(sender_is_receiver),
-            AddKey(_) => cfg.add_key_cost.send_fee(sender_is_receiver),
+            AddKey(AddKeyAction { access_key, .. }) => match &access_key.permission {
+                AccessKeyPermission::FunctionCall(call_perm) => {
+                    let num_bytes = call_perm.receiver_id.as_bytes().len() as u64
+                        + call_perm
+                            .method_names
+                            .iter()
+                            .map(|name| name.as_bytes().len() as u64)
```

Comment:
Add `+1` for every method name (for a terminating character).

Reply:
Done.